src/ontology/edits/oeo-model.omn

```diff
@@ -834,10 +834,10 @@ Class: Minimization
 Class: Model
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A model is a tool for computing the behavior of a system using an idealized representation."@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A model is a generically dependent continuant that is used for computing the behavior of a system using an idealized representation."@en
     
     SubClassOf: 
-        ModelingSoftware,
+        <http://purl.obolibrary.org/obo/BFO_0000031>,
         <http://purl.obolibrary.org/obo/BFO_0000051> some SoftwareElement
     
```

Comment:
Do we use the termtracker just for "big" issues or should it be included here too?

Reply:
I think we should use it as often as possible because it doesn‘t hurt.